activiti-api-impl/activiti-api-process-runtime-impl/src/main/java/org/activiti/runtime/api/impl/ProcessRuntimeImpl.java

```diff
@@ -293,10 +298,12 @@ public void setVariables(SetProcessVariablesPayload setProcessVariablesPayload)
     }
 
     @Override
+    @Transactional
     public void signal(SignalPayload signalPayload) {
         //@TODO: define security policies for signalling
         runtimeService.signalEventReceived(signalPayload.getName(),
```

Comment:
@daisuke-yoshimoto  This code sends the same signal two times.  The `signalEventReceived` method should not be invoked at all. Instead, the `eventPublisher` should simply call `publishEvent(signalPayload)` to broadcast the signal via Cloud Stream signalEvent destination

Reply:
@igdianov
Thanks for your comment.

I also think that it should be only once, but at the time of the first PR, it is the specification that salaboy decided.
Activiti/activiti-cloud-runtime-bundle-service#38